Testscripts/Linux/VERIFY-VHD-PREREQUISITES.py

```diff
@@ -49,7 +49,7 @@ def verify_grub(distro):
         version_release = 0
         if distro == "REDHAT" or distro == "CENTOS":
             version_release = Run("cat /etc/system-release | grep -Eo '[0-9].?[0-9]?' | head -1 | tr -d '\n'")
-        if float(version_release) == 8.0:
+        if float(version_release) >= 8.0:
```

Comment:
This if-statement should be inside of above if-statement (line 50)?

Reply:
Hi @juhlee-microsoft ,
I'm not quite sure about that. It seems that if put it in the if-statement(line50), the distros "ORACLELINUX", "SLES", "FEDORA" will be skipped.

Reply:
She doesn't change the original logic @juhlee-microsoft